go/vt/mysqlctl/mysqld.go

```diff
@@ -1264,19 +1264,23 @@ func (mysqld *Mysqld) OnTerm(f func()) {
 }
 
 func buildLdPaths() ([]string, error) {
-	baseEnv := os.Environ()
```

Comment:
This would break the recently added fix for correctly passing down environment variables like `TZ`.